cime_config/config_grids.xml

```diff
@@ -2683,6 +2718,13 @@
       <desc>SOwISC12to60E2r4 is a MPAS ice/ocean grid with enhanced resolution of 12km in the Southern Ocean around Antarctica. The high resolution regions smoothly transition to the background resolution of the standard low resolution 60to30km grid:</desc>
     </domain>
 
+    <domain name="ECwISC30to60E2r1">
+      <nx>237984</nx>
+      <ny>1</ny>
+      <file grid="ice|ocn">$DIN_LOC_ROOT/share/domains/domain.ocn.ECwISC30to60E2r1.201007.nc</file>
+      <desc>ECwISC30to60E2r1 is a MPAS ocean grid generated with the jigsaw/compass process using the eddy closure density function that is roughly comparable to the pop 1 degree resolution:</desc>
```

Comment:
```suggestion
      <desc>ECwISC30to60E2r1 is a MPAS ocean grid generated with the jigsaw/compass process using the eddy closure density function that is roughly comparable to a uniform 0.5 degree resolution:</desc>
```

For one thing, the comparison to POP seems increasingly irrelevant.  For another, the resolution is about 2x a POP 1 degree grid.  I also wonder if we should be giving some description of where the higher and lower res regions are as the previous EC grids did, but don't have strong feelings about it.

Reply:
Yes this was clearly a "copy and lightly modify from a similar grid" job on my part. I removed the POP reference and added the description of where the higher and lower res regions were. I don't want to refer to this as comparable to a 0.5 degree grid since previous similar grids refer to a comparable 1 degree grid (while neither are really accurate), so I think resolution description is explicit enough for here. Let me know if you agree (or not!).